targets/zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+target_sources(app PRIVATE src/main-zephyr.c src/jerry-port.c src/getline-zephyr.c)
+
+add_library(jerry-core STATIC IMPORTED)
+add_library(jerry-ext STATIC IMPORTED)
+set_target_properties(jerry-core PROPERTIES IMPORTED_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/../obj/lib/libjerry-core.a)
+set_target_properties(jerry-core PROPERTIES INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_CURRENT_SOURCE_DIR}/../../jerry-core/include)
+set_target_properties(jerry-ext PROPERTIES IMPORTED_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/../obj/lib/libjerry-ext.a)
+set_target_properties(jerry-ext PROPERTIES INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_CURRENT_SOURCE_DIR}/../../jerry-ext/include)
```

Comment:
These `set_target_properties` lines are quite long. What about wrapping them?

For example:
```
set_target_properties(jerry-ext PROPERTIES INTERFACE_INCLUDE_DIRECTORIES
                      ${CMAKE_CURRENT_SOURCE_DIR}/../../jerry-ext/include)
```

Reply:
Done